motionplan/tpSpaceRRT.go

```diff
@@ -491,7 +496,7 @@ func (mp *tpSpaceRRTMotionPlanner) getExtensionCandidate(
 		subNode := newConfigurationNode(solution.Configuration[i : i+2])
 
 		// Check collisions along this traj and get the longest distance viable
-		trajK, err := curPtg.Trajectory(subNode.Q()[0].Value, subNode.Q()[1].Value, mp.planOpts.Resolution)
+		trajK, err := curPtg.Trajectory(subNode.Q()[0].Value, 0, subNode.Q()[1].Value, mp.planOpts.Resolution)
```

Comment:
I want to confirm I understand correctly that for the purposes of RRT we are doing IK with a 3DOF PTG and then adding the start value as zero everywhere in this file to make it 4DOF

Reply:
Almost; we really do IK with just 2dof (alpha + dist), and the RRT caller is what has been handling turning that 2dof value into a 3dof value for later use.

We are updating such that the point where the 2dof value would become a 3dof value, now becomes a 4dof value instead.